Makefile

```diff
@@ -1,8 +1,7 @@
 VERBOSE ?= @
-TEST_PROCS ?= $(shell nproc)
 
 SHELL := /bin/bash
-PYTEST := python3 -m pytest -n$(TEST_PROCS) -vv --timeout=3600
+PYTEST := python3 -m pytest -n auto -vv --timeout=3600
```

Comment:
This removes the possibility to set the number of test processes, which is quite useful on systems with a lot of CPU cores. Is that change needed for VS Code?

Reply:
You're right. It's not needed for vscode - I dropped this change altogether.